### ISSUE TITLE
Doubts

### ISSUE THREAD

**Issue by mbenatti** [Voronchuk/elixir_google_spreadsheets#2]
There's a way to get entire worksheet to work with another library like, https://github.com/GrandCru/GoogleSheets ??

An for get the sheet ID's, there's a way to get ids from drive folder using API??

What we need to make this work?

Thanks in advance.

**Comment by Voronchuk** [comment]
We use https://github.com/GrandCru/GoogleSheets then we need only to read the Google Spreadsheet content, so I don't see any problem to run GrandCru library to read spreadsheet which is edited by this library as parallel processes.

As for the sheet id, the files stored on your Google Drive is actually a json files with content like:
```
{"url": "https://docs.google.com/open?id=1w3XXXXXX_6Ed-uKDxkXXXXXxY_hkjh6ikXXXXXXXoxg", "doc_id": "1w3XXXXXX_6Ed-uKDxkXXXXXxY_hkjh6ikXXXXXXXoxg", "email": "voronchuk@gmail.com", "resource_id": "document:1w3XXXXXX_6Ed-uKDxkXXXXXxY_hkjh6ikXXXXXXXoxg"}
```
So it's easy to get sheet ID from it.

**Comment by mbenatti** [comment]
But the point is, I want to read sheets dynamically, not one by one in a json file...

**Comment by Voronchuk** [comment]
I think your best bet is to look on Google Drive REST API: https://developers.google.com/drive/v2/reference/files/list

It's out of scope of this library, never tried this API myself.

**Comment by mbenatti** [comment]
Thank you @Voronchuk 

Another point, 
You can put this library into hex.pm

**Comment by Voronchuk** [comment]
Personally, I don't use multi-sheet documents with Elixir, so we didn't have the need for such operations. This library is trying to map official [Google Sheets API v4](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values) as close as possible:  and there is no `sheet_id` option in the original. 

I have a basic understanding of how to implement such functionality and can look on it over weekend if you really need it, because for read-only functionality GrandCru should suffice.

**Comment by mbenatti** [comment]
@Voronchuk Will be awesome if you can help me with this functionality

**Comment by alex88** [comment]
To select which sheet, use `GSS.Spreadsheet.fetch(pid, "sheet_name!A1:B2")`

as shown in https://developers.google.com/sheets/api/guides/concepts#a1_notation

or, to retrieve a complete sheet, use `GSS.Spreadsheet.fetch(pid, sheet_title)`

**Comment by Voronchuk** [comment]
Implemented in v0.1.2
